packages/debugger/package.json

```diff
@@ -71,6 +71,7 @@
     "@lumino/messaging": "^1.4.3",
     "@lumino/signaling": "^1.4.3",
     "@lumino/widgets": "^1.19.0",
+    "@vscode/debugprotocol": "~1.51.0",
```

Comment:
Looks like this adds `@vscode/debugprotocol` in addition to `vscode-debugprotocol` defined a few lines below.

Maybe we can clean that up to only depend on the new `@vscode/debugprotocol`, as it is the case on `master` now:

https://github.com/jupyterlab/jupyterlab/blob/1903d0dd09b2415a998dd46de15b54a5c8988db5/packages/debugger/package.json#L74

And switch to `^1.51.0` instead of `~1.51.0`

Reply:
Or if `@vscode/debugprotocol` is not used on the `3.3.x` branch, then we can just remove it here and in the `yarn.lock`.

Reply:
For reference the update was done in https://github.com/jupyterlab/jupyterlab/pull/11812

Reply:
I saw it too. I'm on it :wink:

Reply:
Alright thanks!